packages/clima_ui/lib/screens/about_screen.dart

```diff
@@ -39,6 +39,17 @@ class AboutScreen extends StatelessWidget {
                 'https://github.com/CentaurusApps/clima/tree/master/fastlane/metadata/android/en-US/changelogs',
               ),
             ),
+            SettingsTile(
+              title: 'Donate',
+              subtitle: 'Support the development of Clima',
+              leading: Icon(
+                Icons.local_library_outlined,
```

Comment:
I think https://api.flutter.dev/flutter/material/Icons/attach_money-constant.html would be more suitable.